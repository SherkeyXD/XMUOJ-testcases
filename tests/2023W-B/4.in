265185440 330214818 504232720 646772012
