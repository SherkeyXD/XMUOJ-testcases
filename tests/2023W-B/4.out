452860966 181945488 581742634 472754110
